docs/README.md

```diff
@@ -24,3 +24,7 @@ Documents relevant to the CPAN Security Group.
 * Tech & legislation [Reading list](readinglist.md) ⚠️  DRAFT
 * [Glossary](glossary.md) of terms ⚠️  DRAFT
 * [Supply-chain SBOM roles and metadata overview](supplychain-sbom.md) ⚠️  DRAFT
+
+## Other
+
+* [CPAN Dependency Confusion (study)](cpan-dependency-confusion.md) ⚠️  DRAFT
```

Comment:
I've been using the "⚠️  DRAFT" label as a signal for others that the document is a work in progress, and that questions, feedback and corrections are welcome. If you don't mind;

* When you see the "⚠️  DRAFT" label, what comes to your mind? ("This needs help" or "This is not finished"?)
* How useful is the "⚠️  DRAFT" section in the document itself? (links to chat & repository)

Reply:
Yes, was not sure about that. I fixed it